test/e2e/framework/pv/pv.go

```diff
@@ -53,6 +53,10 @@ const (
 
 	// VolumeSelectorKey is the key for volume selector.
 	VolumeSelectorKey = "e2e-pv-pool"
+
+	// IsWindowsStorageClassAnnotation represents a StorageClass annotation that
+	// marks a class as the Windows compatible StorageClass
+	IsWindowsStorageClassAnnotation = "storageclass.kubernetes.io/windows-compatible-class"
```

Comment:
should we prefix it with "e2e." to make it obvious that this is for testing only?

Reply:
added e2e as the prefix